### PR TITLE
use ember-cli

### PR DESCRIPTION
**Pull request by GavinJoyce**
closes https://github.com/eviltrout/ember-performance/issues/24

There are many improvements and simplifications that I'd like to make once this has landed. I purposefully didn't take too much on in this PR though.

For future PRs:

 * Upgrade to Ember 2.1
 * Extract components from controller
 * remove the need for `/index.html` in urls

### CODE REVIEW

**Comment by rwjblue** [comment]
#63 is merged, doesn't look like this needs a rebase though

**Comment by GavinJoyce** [comment]
I just rebased :-)

**Comment by GavinJoyce** [comment]
@rwjblue this is ready for review

**Comment by rwjblue** [comment]
Diff is impossible to skim. I'll try to pull it down locally and review in the morning.

File: app/controllers/index.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
+import Ember from 'ember';
+
+function roundedNumber(num) {
+  if (num) {
+    return Math.round(num * 100, 10) / 100.0;
+  }
+  return num;
+}
+
+var EMBER_PERF_VERSION = "0.9.2"; // TODO: get from package.json
```
Comment (stefanpenner) [inline app/controllers/index.js]: `const`

**Comment by stefanpenner** [comment]
running locally now.

**Comment by stefanpenner** [comment]
seems to work as advertised – LGTM

@GavinJoyce i did notice result numbers are a bit hard to read `317206.03`  Also, metrics such as the 5 number summary might provide some additional insight.

**Comment by stefanpenner** [comment]
Ah, my last comment is likely unrelated to this PR, just for future.

**Comment by stefanpenner** [comment]
this is rad!

**Comment by rwjblue** [comment]
@gavinjoyce - Awesome job!

@stefanpenner - Thanks for pulling it down and reviewing!

**Comment by GavinJoyce** [comment]
Nice one, thanks. I'll follow up with a series of PRs this weekend which will improve the UI and reporting.

**Pull request merged**